<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preference</class>
 <widget class="QWidget" name="Preference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="waterType_check">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fresh water</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Salt water</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>159</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Favorite water type of fish </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="location_input">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Would you like to be anonymous?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="anonymous_check">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="skin_box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Skin 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Skin 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Skin 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Skin 4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Skin preference</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prefSubmit_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fish Buddies</string>
   </property>
  </widget>
  <widget class="QLabel" name="fishIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Private Account?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="private_check">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-300</x>
     <y>-100</y>
     <width>3671</width>
     <height>1216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width:100%;
height:100%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/wind-wave-cartoon-water-waves-6e912dfb99f8506682a88de1e14af64b.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>159</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Favorite water type of fish </string>
   </property>
  </widget>
  <widget class="QComboBox" name="tank_check">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Big tank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Small tank</string>
    </property>
   </item>
  </widget>
  <zorder>label_8</zorder>
  <zorder>waterType_check</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>location_input</zorder>
  <zorder>label_4</zorder>
  <zorder>anonymous_check</zorder>
  <zorder>skin_box</zorder>
  <zorder>label_5</zorder>
  <zorder>prefSubmit_button</zorder>
  <zorder>label_6</zorder>
  <zorder>fishIcon</zorder>
  <zorder>label_7</zorder>
  <zorder>private_check</zorder>
  <zorder>label_9</zorder>
  <zorder>tank_check</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
